### ISSUE TITLE
希望制成安装包

### ISSUE THREAD

**Issue by 1238888888** [microsoft/markitdown#117]
这样才都会用，目前状态不会使用

**Comment by sakurairinkawa** [comment]
？

**Comment by willy62401-work** [comment]
That's it.

1. Install Python from [here](https://www.python.org/downloads/).
2. Once Python is installed, open your terminal or command prompt and run:
```
pip install markitdown
```

**Comment by liguobao** [comment]
This way it can all be used, but the current state cannot be used

What? This is not for you to use

**Comment by neverlatetolearn0** [comment]
> That's it.
> 
> 1. Install Python from [here](https://www.python.org/downloads/).
> 2. Once Python is installed, open your terminal or command prompt and run:
> 
> ```
> pip install markitdown
> ```
After I installed markitdown, I couldn't export MarkItDown

**Comment by chienyuanchang** [comment]
Thanks for reaching out. MarkItDown is a file-to-Markdown converter, so we need a clear description of the problem or the feature you want before we can act on it.

If you'd like to contribute, please open a new issue with a concrete use case, example input, and the result you expect.
